src/blog/2023/10/community-news-10.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Community News October 2023
+subtitle: Your monthly update for the FlowFuse and Node-RED communities
+description: News from the FlowFuse and Node-RED communities
+date: 2023-10-05
+authors: ["ian-skerrett"]
+image: "/images/blog/community-news.jpeg"
+tags:
+    - posts
+    - newsletter
+---
+
+Welcome to the FlowFuse newsletter for October 2023, a monthly roundup of what’s been happening with FlowFuse and the wider Node-RED community. 
+
+<!--more-->
+
+## Quicker Releases for FlowFuse Cloud and New Changelog
+
+FlowFuse is committed to delivery new features to our customers and community as fast as possible. We are now rolling out new builds of FlowFuse to FlowFuse Cloud more regularly than the previous monthly release. This will allow FlowFuse Cloud users to benefit from the lastest enhancements as they become available. The self-hosted version of FlowFuse will continue to have monthly releases.
```

Comment:
```suggestion
FlowFuse is committed to delivery new features to our customers and community as fast as possible. We are now rolling out new builds of FlowFuse to FlowFuse Cloud more regularly than the previous monthly release. This will allow FlowFuse Cloud users to benefit from the latest enhancements as they become available. The self-hosted version of FlowFuse will continue to have releases every 4 weeks.
```